File: src_qt/Node.cpp
#include <QtCore/QString>
#include "Node.h"

namespace CnotiMind
{

	Node::Node(const QString& key, const QString& value):
		_key( key ),
		_value( value )
	{

	}

	const QString& Node::value() const
	{
		return _value;
	}

	const QString& Node::key() const
	{
		return _key;
	}

}

File: src_ios/CnotiMind/Node.m
//
//  Node.m
//  CnotiMind
//
//  Created by Gonçalo Rodrigues on 11/04/21.
//  Copyright 2011 Cnotinfor. All rights reserved.
//

#import "Node.h"


@implementation Node

@synthesize _key = key;
@synthesize _value = value;

- (id) init
{
    if (self == [super init]) {
    }
    return self;
}

//- (id) initWithKeyAndValue:(NSString*)aKey value:(NSString*)aValue
//{
//    if (self == [super init]) {
//        _key = aKey;
//        _value = aValue;
//    }
//    return self;
//}

//  virtual method
- (NSString*) info:(int)depth
{
    return nil;
}


- (void) dealloc
{
    [super dealloc];
}


@end
